mmhealth/types/support_packet.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package types
+
+import "github.com/mattermost/mattermost/server/public/model"
+
+// SupportPacket represents the V1 support packet format (pre-v10.6.0)
+// This struct is copied from github.com/mattermost/mattermost/server/public@v0.0.18
+// to maintain compatibility with older support packets after upgrading to v0.1.21+
+type SupportPacket struct {
```

Comment:
Would that make thinks more clear?
```suggestion
type SupportPacketV1 struct {
```

Reply:
Good suggestion. I have changed this to `SupportPacketV1`.